apps/vmq_enhanced_auth/src/vmq_enhanced_auth.erl

```diff
@@ -954,4 +966,139 @@ simple_acl(_) ->
             split_topic_label(<<"a/b label a b\n">>)
         )
     ].
+add_complex_acl_test(_) ->
```

Comment:
These tests are for complex trie which is not implemented in vmq_enhanced_auth plugin.

Reply:
Removed